modules/fedimint-mint-client/src/backup/recovery.rs

```diff
@@ -614,16 +614,14 @@ impl MintRestoreInProgressState {
 
     pub(crate) fn handle_consensus_item(
         &mut self,
-        epoch: u64,
         peer_id: PeerId,
         item: &ConsensusItem,
         processed_txs: &mut HashSet<TransactionId>,
         rejected_txs: &BTreeSet<TransactionId>,
         secret: &DerivableSecret,
     ) {
         debug!(target: LOG_ECASH_RECOVERY, ?item, "handling consensus item");
-        assert!(epoch == self.next_epoch);
-        self.next_epoch = epoch;
```

Comment:
This assert made it crash when there wasn't a backup, also seemed quite redundant with the following line.